src/utils/Types.ts

```diff
@@ -13,6 +13,7 @@ export type RiddlePreview = {
 	linkId: string;
 	name: string;
 	image?: string;
+	imageFile?: Blob;
```

Comment:
Úplně nevím, jak to předtím fungovalo/nefugovalo, ale přidal jsem si položku pro samotný file (stejně tak u `solvedImage`), abych nemusel přetýpovávat z `Blob` na `string`, jakmile se tam dává ta URL.